corehq/ex-submodules/casexml/apps/phone/restore_caching.py

```diff
@@ -24,7 +23,6 @@ def _restore_payload_path_cache_key(domain, user_id, sync_log_id, device_id):
         domain=domain,
         prefix=RESTORE_CACHE_KEY_PREFIX,
         user_id=user_id,
-        version='2.0',
```

Comment:
Hmm, that's not quite what I had in mind. It's not clear to me why we don't want the sync protocol version in the cache key. Can you help me understand your reasoning here?